### ISSUE TITLE
go-demo-2 crashlooping

### ISSUE THREAD

**Issue by talaniz** [vfarcic/k8s-specs#26]
I'm getting a crashloop on the pod/go-demo-2.yml spec. I'm running a newer Mac so Virtualbox is not a driver I can select, I am instead using docker, not sure if that would make a difference. 

```
❯ kubectl create -f pod/go-demo-2.yml
pod/go-demo-2 created
...
❯ kubectl get -f pod/go-demo-2.yml
NAME        READY   STATUS             RESTARTS      AGE
go-demo-2   1/2     CrashLoopBackOff   5 (10s ago)   55s
...
❯ kubectl logs go-demo-2
Defaulted container "db" out of: db, api
runtime: failed to create new OS thread (have 2 already; errno=22)
fatal error: newosproc

runtime stack:
runtime.throw(0x524da0, 0x9)
        /usr/local/go/src/runtime/panic.go:527 +0x90
runtime.newosproc(0xc82002c000, 0xc82003bfc0)
        /usr/local/go/src/runtime/os1_linux.go:150 +0x1ab
runtime.newm(0x555ce8, 0x0)
        /usr/local/go/src/runtime/proc1.go:1105 +0x130
runtime.main.func1()
        /usr/local/go/src/runtime/proc.go:48 +0x2c
runtime.systemstack(0x5c4300)
        /usr/local/go/src/runtime/asm_amd64.s:262 +0x79
runtime.mstart()
        /usr/local/go/src/runtime/proc1.go:674

goroutine 1 [running]:
runtime.systemstack_switch()
        /usr/local/go/src/runtime/asm_amd64.s:216 fp=0xc820026770 sp=0xc820026768
runtime.main()
        /usr/local/go/src/runtime/proc.go:49 +0x62 fp=0xc8200267c0 sp=0xc820026770
runtime.goexit()
        /usr/local/go/src/runtime/asm_amd64.s:1696 +0x1 fp=0xc8200267c8 sp=0xc8200267c0
```

**Comment by vfarcic** [comment]
This is a repo I haven't used (or maintained) for a few years now.

Can you please let me know how you got to it? Are you following some of my books/videos/tutorials?

**Comment by talaniz** [comment]
Sorry, I must have missed the github notification. I'm following the DevOps Toolkit 2.3

**Comment by vfarcic** [comment]
That's a very old book. I stopped maintaining the repo used in it and removed the book from the shelves (at least those I have control of). Books are not very flexible and hard to maintain and I stopped writing them. Instead, now I focus on YouTube videos which are smaller in scope, less likely to go out of date, and easier to keep up-to-date. You'll find both beginners and more advanced topics around Kubernetes in https://youtube.com/@DevOpsToolkit.
